packages/core/src/utils/behaviors.js

```diff
@@ -0,0 +1,27 @@
+import { isEscapeKeyEvent } from './helper-functions';
+
+export class CloseOnEscBehavior {
+	constructor(parentElement) {
+		this.parentElement = parentElement;
+		this.onKeyDown = this.onKeyDown.bind(this);
+	}
+
+	start() {
+		document.addEventListener('keydown', this.onKeyDown);
+	}
+
+	stop() {
+		document.removeEventListener('keydown', this.onKeyDown);
+	}
+
+	onKeyDown(event) {
+		if (
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s) in this "if" block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACurlyBraces)